src/app/shared/components/login.html

```diff
@@ -1,6 +1,8 @@
 <sd-small-modal title="Login" #modal>
   <div *ngIf="failure && userLogin" class="alert alert-danger">
     Benutzername oder Password sind ungültig.
+    <br>
+    <a href="https://my.rabe.ch/auth/forgotpw">Passwort zurücksetzen?</a>
```

Comment:
Shouldn't that be made configurable instead of having a hard coded RaBe URL in the markup?

Reply:
We do not have that built yet, but it should be possible: https://imran3.medium.com/runtime-configuration-for-angular-c9d9082e1de3
If it does not hurry, I can look into it next week.

Reply:
We have references to RaBe allover the markup and it would be quite arbitrary to move them all to some configurable settings. I suggest do wait with this work until there is another organization using raar-ui.